src/v2i-hub/TimPlugin/src/XmlCurveParser.h

```diff
@@ -0,0 +1,50 @@
+#ifndef XML_CURVE_PARSER_H
```

Comment:
Can we rename to TIM_Parser for the time being till we implement the web service part to receive TIM messages

Reply:
This will require more time as variables are currently in use. I will mark this down as an issue and we may be able to pick this up in the upcoming sprint.